app/src/main/java/org/fossasia/susi/ai/chat/STTfragment.kt

```diff
@@ -29,6 +32,11 @@ class STTFragment : Fragment() {
     lateinit var recognizer: SpeechRecognizer
     lateinit var chatPresenter: IChatPresenter
     private val thisActivity = activity
+    private var textToSpeech: TextToSpeech? = null
+    private lateinit var runnable: Runnable
+    private lateinit var delayRunnable: Runnable
+    var mainHandler: Handler = Handler()
+    var subHandler: Handler = Handler()
```

Comment:
Where are you changing this variable?

Reply:
And from which external class are you accessing these variables?

Reply:
> And from which external class are you accessing these variables?

This is prompting up because I reset the head 1 commit behind. The variables of handlers were already defined there